### ISSUE TITLE
Duda entrega 2, grupo impar relacion tour y eventos

### ISSUE THREAD

**Issue by Franbelen** [IIC2413/Syllabus-2022-2#116]
Hola buena analizando los datos, creo que no hay manera de relacionar un evento con el tour que pertenece, ya que este dato no se encuentra en ni una tabla de las ingresadas (como grupo suponemos que un tour se conforma de varios eventos), por lo que el evento debería tener a que tour pertenece o el tour cuales eventos contiene pero no se encuentra ni uno de estos

**Comment by IgnAMS** [comment]
Fui a revisar los tours, y los eventos que están en un tour tienen el mismo nombre que el evento.

**Comment by franancic** [comment]
efectivamente, los tours tienen el mismo nombre que el evento, de esa forma pueden relacionarlo:)

**Comment by tzehnder** [comment]
Entonces un tour puede consistir de muchos eventos los cuales tienen el mismo nombre pero distinta fecha?

**Comment by franancic** [comment]
Si

**Comment by Diegofpdt** [comment]
En ese caso, existen una cantidad considerable de eventos que se supone pertenecen a un tour pero este no existe en el archivo de tours, por ejemplo el BZRP Tour no aparece en el archivo de tours.
Por lo anterior tengo las siguientes dudas:

1.Dado esto, por ejemplo en la consulta número 4 de los grupos impares, asumo que no tendríamos que considerar estos tours que no aparecen en el archivo tour.

2.También en la consulta número 3, solo consideramos los tours que se encuentran en el archivo tour, dado que no tendríamos la información de los otros tours.

**Comment by franancic** [comment]
Si **no** aparecen en el archivo tours, entonces **no** es un tour. 
En casi de BZRP Tour, es alcance de nombre nomas (en donde van diferentes artistas, pero no es Tour).
